### ISSUE TITLE
loss is more and more high

### ISSUE THREAD

**Issue by ZHEQIUSHUI** [awslabs/autogluon#190]
>[Epoch 0] Training cost: 35.768, ObjLoss=42.200, BoxCenterLoss=37.298, BoxScaleLoss=14.781, ClassLoss=0.939
[Epoch 0] Validation: person=0.6338410132892208 mAP=0.6338410132892208
[Epoch 1] Training cost: 17.886, ObjLoss=46.407, BoxCenterLoss=37.463, BoxScaleLoss=14.374, ClassLoss=1.023
[Epoch 1] Validation: person=0.0 mAP=0.0
[Epoch 2] Training cost: 31.236, ObjLoss=52.590, BoxCenterLoss=37.873, BoxScaleLoss=14.191, ClassLoss=1.185
[Epoch 2] Validation: person=0.0 mAP=0.0
[Epoch 3] Training cost: 29.734, ObjLoss=54.700, BoxCenterLoss=37.806, BoxScaleLoss=13.992, ClassLoss=1.190
[Epoch 3] Validation: person=0.0 mAP=0.0
[Epoch 4] Training cost: 22.907, ObjLoss=63.392, BoxCenterLoss=38.285, BoxScaleLoss=14.898, ClassLoss=1.272
[Epoch 4] Validation: person=0.0 mAP=0.0
[Epoch 5] Training cost: 19.434, ObjLoss=72.727, BoxCenterLoss=38.863, BoxScaleLoss=14.923, ClassLoss=1.251
[Epoch 5] Validation: person=0.0 mAP=0.0
[Epoch 6] Training cost: 26.812, ObjLoss=78.407, BoxCenterLoss=39.377, BoxScaleLoss=15.238, ClassLoss=1.155
[Epoch 6] Validation: person=0.0 mAP=0.0
[Epoch 7] Training cost: 27.619, ObjLoss=82.524, BoxCenterLoss=39.706, BoxScaleLoss=15.594, ClassLoss=1.060
[Epoch 7] Validation: person=0.0 mAP=0.0

**Comment by zhanghang1989** [comment]
Looks like the training diverges. Which dataset are you using?

**Comment by zhanghang1989** [comment]
@chongruo do you have some insight on this issue?

**Comment by chongruo** [comment]
It is probably because you are in a trial with a large learning rate. Could you try to lower the range of searching space for the learning rate?

**Comment by ZHEQIUSHUI** [comment]
it is a custom daatset about human head,sometime the result is pretty good but sometime is very bad(same code in two training process),....the mAP was sudden death.


this is my dataset.you can test it if you have free time.i can upload to google drives if you need.thank you
>https://pan.baidu.com/s/1LnxMH0auYR6SzWWelXBo5g  密码: ci41

i don`t know it is my wrong or bug.the code couldn`t save model,and i haven`t saw the argument to choose yolo or ssd or fasterrcnn.
i guass @chongruo mean is set 'ag.Categorical(5e-4, 1e-4)'?
this is my original code(amazing! i just need two lines code!):
>import autogluon as ag
from autogluon.task.object_detection.object_detection import ObjectDetection as task
dataset_train = task.Dataset('/home/arno/Dataset/SCUT_HEAD_Part_B', classes=('person',))
time_limits = 5 * 60 * 60  # 5 days
epochs = 30
detector = task.fit(dataset_train,num_trials=2,epochs=epochs,lr=ag.Categorical(5e-4, 1e-4),ngpus_per_trial=1,time_limits=time_limits, batch_size=8,save_prefix='head_',save_interval=1)

**Comment by chongruo** [comment]
Yes, you can set small learning rates by changing parameters in ag.Categorical(). We will support ssd and faster-rcnn soon.

**Comment by ZHEQIUSHUI** [comment]
thank you for your help, i have tried( ag.Categorical(1e-4,1e-5,1e-6 )),the mAP always got 0.0.when i use demo lr(5e-4,1e-4),sometime it can got a good result....so i don`t know what is the reason for this problem.

**Comment by chongruo** [comment]
Is your final mAP 0.0, or just intermediate mAP 0.0? Did you finish training all trials? It will final_fit the model after obtaining the best config.

**Comment by ZHEQIUSHUI** [comment]
maybe i was too anxious...i will try to finish the whole process,i will put issue if i have new problem,thank you and sorry.
